paymentsheet/src/main/java/com/stripe/android/paymentsheet/state/PaymentSheetLoader.kt

```diff
@@ -221,11 +221,10 @@ internal class DefaultPaymentSheetLoader @Inject constructor(
         return paymentMethods.filter { paymentMethod ->
```

Comment:
I believe `getPaymentMethods` above has to make an API request for *each* saved payment method type. We shouldn't make API requests for SEPA or any other PM type except card and US bank account.

I think that means changing something in `getSupportedSavedCustomerPMs` so that it only returns cards and US Bank accounts (or maybe just rewriting line 205 to be `val paymentMethodTypes = [card, us bank account]`).

Reply:
That's a good callout! Seems like a nice optimization to throw in with this change.